examples/nextjs-full-stack/README.md

```diff
@@ -45,7 +45,7 @@ MIRO_REDIRECT_URL=http://localhost:3000/api/redirect/
 
 5. Run `yarn dev` to start the local server
 
-Once your server is up and running, navigate to [http://localhost:3000/](http://localhost:3000/) in your browser.
+Once your server is up and running, open a board in your developer team and start the app by clicking on your App icon.
```

Comment:
```suggestion
Once your server is up and running, go to [Miro.com](https://miro.com), open a board in your developer team and start the app by clicking on your App icon in the toolbar on the left.
```

Reply:
Mettin's suggestion is the only change I'd suggest 👍